content/blog/2022/2022-04-20-KaplanMeier.md

```diff
@@ -0,0 +1,73 @@
+
+![](/orange-hugo/static/blog_img/2022/2022-04-20-KM-calculatingmanually.png)
+
+On the 5th and 6th day none of the remaining flowers withered but two got censored. There is no need to calculate probability of survival on days where there are only censored data points, but for the sake of clarity we included the calculations in the table. We can see that the censored data affects the number of flowers at risk but does not change the survival probability. On the eight day there are thus only 2 flowers at risk and one of them withers. Taking into account the survival probability of the previous day this gives us the probability of survival of 0.24.
+
+What would happend if we would ignore censored data? We could just exclude them from the analysis but we don't want to do that, because these data points are still informative when it comes to survival. And if we would treat them as surviving we would overestimate the survival probability. 
+
+If we plot the survival probability for each day we get the Kaplan-Meier Plot. 
+
+
+**Orange usecase**
+
+Now that we calculated the Kaplan-Meirer estimator on a small dataset manually and got an understanding what's going on, let's perform this on a larger dataset in Orange. We will use the data from the German Breast Cancer Study Group 2, a survival dataset that is made readily available in Orange. First we load the data using the **Datasets** widget and inspect the data in tabular format with the use of **Data Table**. We see that our data consists of 686 data instances that stand for patients. Each patient is characterized by 8 clinical features and the time at which reacurrence of the disease happened or when the patient stopped being part of the experiment due to unknown reasons (censored). We then connect our data with the **Kaplan-Meier widget** that calculates the survival probability at different time points and plots the survival curve. The widget also allows us to choose one of the three categorical features and compare the survival curves of the different categories.
+
+![](/orange-hugo/static/blog_img/2022/2022-04-20-kmworkflow.png)
+
+Comparing two or more survival curves can be done by visual inspection or by comparing different statistics. For instance we can compare the survival curves of patients according to their tumor grade which is a quantative measure of how abnormal the tumor cells and tissue look under a microscope. For breast cancer doctors most often use the Nottingham grading system which divides the tumor into three grades, where 1 coresponds to low grade, 2 to intermediate grade and 3 to high grade. The high grade means that the cells and tissue look most abnormal compared to healthy tissue. 
```

Comment:
For breast cancer doctors most often use --> Breast cancer doctors most often use ?
where 1 coresponds --> where 1 corresponds